libs/libarchfpga/src/read_fpga_interchange_arch.cpp

```diff
@@ -2103,6 +2252,16 @@ struct ArchReader {
                 grid_def.loc_defs.emplace_back(std::move(single));
             }
 
+            // The constant source tile will be placed at (0, 0)
```

Comment:
How are we sure that there is no real tile at (0, 0) ?

Reply:
There is an empty outer layer surrounding the grid, as all the tiles have been placed with a `(1, 1)` offset: https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/libs/libarchfpga/src/read_fpga_interchange_arch.cpp#L2095-L2096